public/partials/report/modals/filter-prompt-modal.html

```diff
@@ -32,6 +32,36 @@ <h4 class="modal-title" translate>Runtime Filter properties</h4>
                 </div>
             </div>
         </div>
+
+        <div class="row">
+
+            <div class="col-md-8">
+                <label class="control-label" translate>Suggestions</label>
+                <div class="checkbox popover-primary ng-scope" popover-trigger="mouseenter" popover-placement="top" popover="{{ 'Select if you want suggest values on filter' | translate }}">
+                    <div>
+                        <label>
+                            <input type="checkbox" ng-model="selectedFilter.prefillEntries.show">
+                            <span translate>Show suggestions</span>
+                        </label>
+                    <span ng-if="selectedFilter.prefillEntries.valuesCount > 0">({{selectedFilter.prefillEntries.valuesCount}})</span>
+                    </div>
+                    <div ng-if="selectedFilter.prefillEntries.show && !selectedFilter.prefillEntries.unlimited">
+                        <label>
+                            <span translate>Number of suggestions</span> 
+                            <input type="number" ng-model="selectedFilter.prefillEntries.limit" placeholder="{{selectedFilter.prefillEntries.limit}}">
```

Comment:
It's possible to enter a negative value, 0, or an empty value. In those cases, all suggestions will be returned even if the "unlimited" option is not enabled.
If the value entered is invalid, the limit should default to a sane value (probably 15, the actual default value)

Reply:
I made some conditions to check to validate the limit